tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/ContextTckResource.java

```diff
@@ -345,7 +345,7 @@ public Response leave(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) String lraId) {
         return Response.ok().entity(lraId).build();
     }
 
-    @PUT
+    @GET
```

Comment:
This is is quite a big change to the spec so I would advise that we at least include a comment, something like:

> @GET // @PUT SHOULD be used but verify that other HTTP methods are supported on compensations (and ditto for the completion, status and forget callbacks).